app/services/early_career_teachers/create.rb

```diff
@@ -17,7 +17,7 @@ def call
         end
 
         ParticipantProfile::ECT.create!({ teacher_profile: teacher_profile, schedule: Finance::Schedule.default }.merge(ect_attributes)) do |profile|
-          ParticipantProfileState.create!({ participant_profile: profile })
+          ParticipantProfileState.find_or_create_by!(participant_profile: profile)
```

Comment:
Why? It should only be possibly to create, since it's in the create block for the profile.

Reply:
Ah, a global find and replace is why :P